src/main.swift

```diff
@@ -1,13 +1,21 @@
 import AppKit
 import Darwin
 
+func printStackTrace() {
+    let stackSymbols = Thread.callStackSymbols
+    for symbol in stackSymbols {
+        print(symbol)
+    }
+}
+
 // - if the app is quit/force-quit from Activity Monitor, it will receive SIGTERM and applicationWillTerminate won't be called
 // - if the app crashes in swift code (e.g. unexpected nil object), SIGTRAP is sent
 // we intercept these signals, and reset the native command-tab shortcut
 [SIGTERM, SIGTRAP].forEach {
     signal($0) { s in
         setNativeCommandTabEnabled(true)
         debugPrint("Exiting after receiving signal", s)
+        printStackTrace()
```

Comment:
I tried to test this, and somehow this signal catching function seems to no longer work. I'm surprised because it worked in the past.

Maybe swift changed or macOS changed. Something seems to have broken and now I can't see any print here.

Reply:
![QQ_1728722754252](https://github.com/user-attachments/assets/442d9896-7219-4d47-8e55-8514c8f7a25a)

I can show it here, adding this allows users to give us some error traces

Reply:
Were you able to show this in the debug build? Could you share the steps? I'm not able to do this locally on my debug build

Reply:
![QQ_1728726415927](https://github.com/user-attachments/assets/27e01308-24b4-4a50-a1ad-1e40d11f76f3)

Yes, I am in the debug build. I add the followed code to App.swift on app start up.

```swift
        let a = [1]
        print(a[5])
```

Reply:
I was able to get the same result as you with this particular way 👍 

What I can't get to work, that I know worked before, is to send signals from outside the app. Like `ctrl+C`, or `kill pid-of-alttab`, or stopping the debugger.

I see the comment I left in the past `if the app is quit/force-quit from Activity Monitor, it will receive SIGTERM`. I tried quitting from Activity Monitor and it doesn't trigger it:

<img width="966" alt="image" src="https://github.com/user-attachments/assets/430c60fb-e143-4f2b-a753-c6d15c544364">

Anyway, it's not related to the issue and this PR. It's a general observation. We can look into it another time 👍